Week3/scripts/3.js

```diff
@@ -0,0 +1,122 @@
+function drie(){
+    console.clear();
+
+    function sum_up(num1, num2, num3){
+        return (num1+num2+num3);
+    }
+
+    console.log(sum_up(3,7,9));
+
+    function colorCar(clr){
+        console.log("a " + clr + ' car')
+    }
+
+    colorCar('red');
+
+    function show_pairs(obj){
+        console.log(Object.entries(obj));
+    }
+    
+    cars = {one: "BMW", two: "TESLA", three:"TOYOTA" }
+    show_pairs(cars);
+
+    function vehicleType(color, code){
+        if (code === 1){
+            console.log("a " + color + ' car');
+        }else if(code === 2){
+            console.log("a " + color + ' motorbike');
+        }else{
+            console.log("You entered a wrong code. Please try again!");
+        }
+    }
+
+    vehicleType("blue", 2);
+
+    console.log((3===3) ? "Yes" : "No");
+
+    function vehicle(col, code, age){
+        if(code != 1 && code !=2){
+            console.log("Please enter a valid code number!");
+            return;
+        }
+        console.log("a " + col + ((age<3)? " new":" used") + ((code===1)? " car": " motorbike"));
```

Comment:
Ah, compact solution! It's not easy to read, so you can make it more readable if you want, but the solution itself is fine